medusa/storage/azure_blobs_storage/azcli.py

```diff
@@ -54,7 +52,7 @@ def cp_upload(self, *, srcs, bucket_name, dest, max_retries=5):
         # Az cli expects the client to provide the MD5 hash of the upload
         for src in srcs:
             cmd = self._az_cli_cmd + ["storage", "blob", "upload", "-f", str(src), "-c", bucket_name, "-n", dest,
-                                      "--content-md5", AbstractStorage.generate_md5_hash(src)]
+                                      "--overwrite", "--validate-content"]
```

Comment:
Does `--validate-content` cause the md5 checksum to get computed for each chunk? If so, isn't that what we're trying to avoid?

Reply:
I can remove this as it's additional cpu/memory load, but since we're invoking the azcli using a subprocess, memory should be freed as soon as the upload finishes 😕 
I'll amend the PR to remove it nonetheless, md5 checks can be enabled in config anyway.